lib/ansible/modules/web_infrastructure/acme_certificate.py

```diff
@@ -34,12 +34,16 @@
       C(dns-01) the necessary dns record has to be created.
       It is I(not) the responsibility of this module to perform these steps."
    - "For details on how to fulfill these challenges, you might have to read through
-      U(https://tools.ietf.org/html/draft-ietf-acme-acme-12#section-8).
+      L(the specification,https://tools.ietf.org/html/draft-ietf-acme-acme-12#section-8).
```

Comment:
I am not aware of a L() format option. Did I miss anythin? https://docs.ansible.com/ansible/latest/dev_guide/developing_modules_documenting.html#formatting-functions

Reply:
Interesting. It has been around for some time: https://github.com/ansible/ansible/blob/devel/docs/bin/plugin_formatter.py#L80
I've seen that while working on some formatting problems, and assumed I could simply use it :-) (actually, I think I already used it in at least on of the `acme_*` before)

Reply:
nice! (and uncool) that this is not documented (even though I could hardly found the docs)

Reply:
Actually, when displaying the docs with `ansible-doc`, `L(...,...)` doesn't work. No idea how the man files behave (can't check right now).

Reply:
It has been introduced in #37009, and it apparently was only meant (back then) for relative links.

Reply:
@gundalow, since you introduced that feature: is it acceptable to also use it like in here (to link to external pages more nicely)?

Reply:
Yes, please use it.